### DISCUSSION TITLE
Add a skip_validation flag to the from_orm method

### DISCUSSION THREAD

**Discussion by blazing-gig** [pydantic/pydantic#2846] [Feature Request]
Hey, 

So after playing around with Pydantic in a couple of normal python projects, I started using it in my Django project with the help of this relatively new framework named [django-ninja](https://django-ninja.rest-framework.com/) which uses Pydantic for serialization / de-serialization. 

Upon writing code, I found that the `.from_orm` method performs model validation **every time**. I also found that there's a method named `.construct` which allows for model instance creation skipping validation to boost performance.

I'd like to propose the same be done for the `.from_orm` method as well to get better performance while serializing `model.Model` objects to Pydantic instances. This use-case would be widely prevalent while performing reads from the DB as the data coming out is assumed to be right and validated.

An example would be:

```py
@org_api_router.get('/', response=List[OrgSchemaOut])
def org_list_view(request):
    qs: QuerySet[Org] = Org.objects.all()
    data: List[OrgSchemaOut] = [
        # OrgSchemaOut.from_orm(org) for org in qs  <---- This is the current impl
        OrgSchemaOut.from_orm(org, skip_validation=True) for org in qs  <---- This is the proposed impl
    ]
    return data
```

If the idea can be agreed upon, I could send a PR for the same.

**Comment by eddsalkield** [comment]
There's an open issue on this topic at the moment, at https://github.com/samuelcolvin/pydantic/issues/1212.  There was willingness to accept a PR for this last year, but it appears that progress has since stalled.  Maybe you can be the one to write it? :slightly_smiling_face:
